dali/operators/decoder/audio/audio_decoder_test.cc

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+#include <fstream>
+#include <string>
+#include "dali/core/format.h"
+#include "dali/operators/decoder/audio/generic_decoder.h"
+#include "dali/test/dali_test_config.h"
+
+namespace dali {
+namespace {
+std::string audio_data_root = make_string(testing::dali_extra_path(), "/db/audio/wav/");  // NOLINT
+
+
+template<typename T>
+std::vector<T> file_to_vector(const std::string &filepath) {
+  std::ifstream file(filepath.c_str());
```

Comment:
Wont' this will read the file as text? You should open it as `ios::in | ios::binary`

Reply:
Yes, and that's what I wanted here. The file contains decoded raw PCM data, so I want to compare this one to the one that's decoded with AudioDecoder

Reply:
It took me a while to see the difference between `file_to_vector` and `file_to_bytes` which also reads to a vector. Why not something like `ReadBytesFromText` and `Read` (or similar)?